store/policy.go

```diff
@@ -574,3 +574,131 @@ func (s *Store) upsertPolicyCache(policyType api.PolicyType, resourceID int, pay
 	}
 	return nil
 }
+
+// PolicyMessage is the mssage for policy.
+type PolicyMessage struct {
+	UID               int
+	ResourceID        string
+	ResourceType      api.PolicyResourceType
+	Payload           string
```

Comment:
(nit: I'm thinking of introducing a new oneof proto so that the caller doesn't need to do much of proto/json/string conversions. But, later, not this PR.)

https://developers.google.com/protocol-buffers/docs/proto3#oneof